modules/elasticsearch-index/src/main/java/org/opencastproject/elasticsearch/index/theme/IndexTheme.java

```diff
@@ -51,7 +58,7 @@
         "licenseSlideDescription", "licenseSlideBackground", "watermarkActive", "watermarkFile", "watermarkPosition" })
 @XmlRootElement(name = "theme", namespace = IndexObject.INDEX_XML_NAMESPACE)
 @XmlAccessorType(XmlAccessType.NONE)
-public class Theme implements IndexObject {
+public class IndexTheme implements IndexObject {
```

Comment:
Somehow I don't like the IndexTheme renaming. I fully understand why you rename it and I think its better than before but I'm not sure if IndexedTheme would not be a better name.

Reply:
Ha, I couldn't decide between those two! Maybe I'll change that later...

Reply:
Ha, I couldn't decide between those two!

By the way: Series and Events do not have this issue! That's because they are representing data from different services, while the IndexTheme is a 1:1 representation of a Theme object, thus the naming problem.

The longterm plan is to stop indexing Themes anyway, since it doesn't make much sense, but this was out of scope for now. So this problem will hopefully resolve itself.